docs/changelog/103319.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 103319
+summary: Store `semantic_text` model info in mappings
```

Comment:
Question, maybe for @carlosdelest: Do we want changelog PRs for this type of change, or should this be tagged as `>non-issue` in favor of a future release highlight PR?

Reply:
IMO it makes sense to remove the changelog, especially since we are merging into a feature branch now

Reply:
let's use >non-issue for all of these changes until we merge to main / do the final PR there 👍